upgrading/topics/keycloak/changes.adoc

```diff
@@ -2,6 +2,27 @@
 
 === Migrating to 11.0.0
 
+==== Upgrade to Wildfly 20
+
+The {project_name} server was upgraded to use Wildfly 20 as the underlying container. This does not directly involve any
+specific {project_name} server functionality, but there are few changes related to the migration, which worth mentioning.
+
+Dependency updates::
+  The dependencies were updated to the versions used by Wildfly 20 server. For example, Infinispan is now 10.1.8.Final.
+
+Configuration changes::
+  There are few configuration changes in the `standalone(-ha).xml` and `domain.xml` files. You should follow the <<_install_new_version>>
+  section to handle the migration of configuration files automatically.
+
+Cross-Datacenter Replication changes::
+  * You will need to upgrade {jdgserver_name} server to version {jdgserver_version}. The older version may still work, but it is
+  not guaranteed as we don't test it anymore.
+  * It is recommended to use `protocolVersion` property added to the `remote-store` under configuration of infinispan caches. When
```

Comment:
Maybe replace

    It is recommended to use `protocolVersion` property added to the `remote-store` under configuration
    of Infinispan caches.

with

    It is recommended to use the `protocolVersion` attribute of the `remote-store` element when
    configuring Infinispan caches.

?

Reply:
Thanks. I will use the 
```
It is recommended to use the `protocolVersion` property added tp the `remote-store` element when configuring Infinispan caches.
```
It is technically "property" element, not attribute of the "remote-store" element.